listenbrainz/webserver/templates/login/login.html

```diff
@@ -8,16 +8,21 @@ <h2 class="page-title">Sign in</h2>
   <p>To sign in, use your MusicBrainz account, and authorize ListenBrainz to access your profile data.</p>
 
   <div class="well bg-danger" style="max-width: 600px; margin: 0 auto;">
-      <p class="text-danger">Important!</p>
-      <p>
-        By creating a ListenBrainz account, you acknowledge that your listen data will be made available to the public
-        and that the public may download the entirety of the ListenBrainz data under the <a
-        href="https://creativecommons.org/publicdomain/zero/1.0/">CC0 license</a>.
-      </p>
-  </div>
-  <br/>
-  <div class="well" style="max-width: 600px; margin: 0 auto;">
-    <a href="{{ url_for('login.musicbrainz', next=request.args.get('next')) }}"
-       class="btn btn-primary btn-lg btn-block">Sign in with MusicBrainz</a>
-  </div>
+    <p class="text-danger">Important!</p>
+    <p>
+      By creating a ListenBrainz account, you grant the MetaBrainz Foundation permission to process your listen history (not your personal data from your profile) into data dumps we make publicly available under the <a
```

Comment:
"by creating an account" conflicts with "no, take me to the delete page" below. Is it possible to make a distinction between new and returning users?